pkg/client/cluster-essentials/pkg/fake/fakes.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2023 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package fake contain cluster essential util APIs
+package fake
```

Comment:
I don't think there is a need for a separate `fake` package. Just move this file to the `client` package alongside the install files.

Reply:
Please https://github.com/vmware-tanzu/tanzu-framework/pull/4503#discussion_r1157914392 for desired package structure.